code/game/objects/items/choice_beacon.dm

```diff
@@ -175,3 +175,17 @@
 	GLOB.holy_armor_type = choice_path
 	return ..()
 
+/obj/item/choice_beacon/hos
+	name = "gun beacon"
+	desc = "A single use beacon to deliver a gunset of your choice to help with security detail."
+	company_source = "Central Command"
+	company_message = span_bold("Supply Pod incoming, please stand back.")
+
+/obj/item/choice_beacon/hos/generate_display_names()
+	var/static/list/selectable_guns = list(
+		"X-01 MultiPhase Energy Gun" = /obj/item/gun/energy/e_gun/hos,
+		"Lawbringer" = /obj/item/gun/energy/e_gun/lawbringer,
+		"Combat Shotgun" = /obj/item/gun/ballistic/shotgun/automatic/combat,
```

Comment:
Also if this is literally just a normal shotgun then yeah dont include it, it should be unique in SOME way, if you can find/make a unique shotgun then ill think about it

Reply:
Included wrong shotgun, its now compact type meaning it can go on waist.